src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -30,14 +35,12 @@
     bottom: -1px;
     right: -1px;
     border-radius: 50%;
-    height: 8px;
-    width: 8px;
     z-index: z(user-avatar-status-indicator);
     display: block;
+    @include square-size(8px);
```

Comment:
I think the indicator should be dependent of the container size and should be defined in %.

Reply:
The % is exactly what causes the issue, even when no modifier is provided.

Reply:
But, let's see if there is some reasonable `px` for all 3 modifiers

Reply:
@mike-lvov considering: 

```scss
  &__status-indicator {    

    // .....

    @include square-size(8px);

    &--large {
      @include square-size(12px);
    }
  }
```


The status  indicator for all but `--large` is `8px` . Do you think there is a necessity of defining value for `--small` and `--medium` other than the default?

Reply:
@goran-peoski-work I don't think I understand what you mean by 

> The % is exactly what causes the issue, even when no modifier is provided

Setting the % value for the `indicator` should not cause any issue or am I missing something?

Reply:
Well, the difference of 4 pixels - default at 8px, only set to 12px when size set to large... There wasn't any benefit of playing with % in that interval, but I did rework how the size is set, defaulting to the current a.k.a. legacy if not set.

```scss
  $avatar-sizes-map: (
    // NOTE: legacy not added because it is the default
    small: 25px,
    medium: 40px,
    large: 54px,
    full: 100%
  );
  $indicator-sizes-map: (
    // NOTE: legacy not added because it is the default
    small: 8px,
    medium: 10px,
    large: 12px,
    full: 25%
  );
```

Reply:
NOTE: the legacy behavior is still set at 8px, not scaling with %, and will have to get some one who knows the possibility of that causing issues to point it out for me to test.

Reply:
this is the current state b40bc48846f23c0b8cebb8e9f7f5941bfc5b29bf

Reply:
@goran-peoski-work just setting the size of indicator to 25% should simplify the thing, I don't understand what's the stopping point here. It will automatically be adjusted to any size changes & if a person will set the size via the classes for ex.

Reply:
Btw, the current `legacy` doesn't look right
![image](https://user-images.githubusercontent.com/63194656/126531987-921b170e-5885-4aba-86bf-2b49d8b0dbbd.png)

Reply:
> I don't understand what's the stopping point here.

@mike-lvov OOS?

An issue about the UserAvatar not being square is morphing to issue about the indicator. And I'd be OK fixing it if it was all about simply about slapping a % and saying it's done. 

It is not a static proportion, it is granular.
At `25px` and at `48px` the indicator is both `8px`, that's `32%` and `16%` respectively, while for the large size - `22%`.

So instead of us here trying to figure out what looks right in which cases, I suggest we ask design to give us proper guide on that, we create separate PR for the indicator and until then, leave the legacy indicator as it already was and just fix the avatar size in this PR while providing a manageable interface for future addition/expansion.

There are after all 2 other PRs relating to different aspects of the avatar, so one more shouldn't be that of a problem.

Reply:
@goran-peoski-work okay, I agree that it's a design issue